albx/session02/fizz_buzz.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+#Write a function that prints the numbers from 1 to 100 inclusive but
+#  for multiples of three print "Fizz" instead of the number
+#  for multiples of five print "Buzz"
+#  for multiples of both three and five print "FizzBuzz"
+#http://uwpce-pythoncert.github.io/IntroToPython/exercises/fizz_buzz.html#exercise-fizz-buzz
+
+
+input = [x for x in range(1,101)]
+for i in input:
```

Comment:
`input` is a protected keyword in python so you should choose a different variable name.  Using words that are part of the standard library functions are allowed in python, but can cause some weird results